lib/ama_validators/name_format_validator.rb

```diff
@@ -1,6 +1,6 @@
 class NameFormatValidator < ActiveModel::EachValidator
   def validate_each(object, attribute, value)
-    unless value =~ /\A[\sa-zA-ZÀàÂâÄäÈèÉéÊêËëÎîÏïÔôŒœÙùÛûÜüŸÿÇç,.'-]+\z/
+    unless value =~ /\A[\sa-zA-ZÀàÂâÄäÈèÉéÊêËëÎîÏïÔôŒœÙùÛûÜüŸÿÇç,.'\-\)\(]+\z/
```

Comment:
If you add `i` after the last `/`, it'll be case-insensitive, so you could get rid of half of those characters.